backend/src/controllers/encounter.controller.ts

```diff
@@ -16,11 +16,8 @@ import {getUserByAuthId} from "../services/user.service";
    logger.info("POST /encounter request from frontend");
  
    try {
-     // Grab the data from the req
-     const encounterReq = getEncounterFromReqBody(req.body);
-
      // Pass data to service and attempt to save
-     const createdEncounter = await encounterService.createEncounter(encounterReq);
+     const createdEncounter = await encounterService.createEncounter(req.body);
```

Comment:
Are we adding a reference to this `encounter` to the `user` and `person` that owns it? I can't seem to find that.

Reply:
@AdamAWiener sorry think I messed up from a previously merged issue. I think this createEncounters() was merged from the issue (#94). Currently @schi314 is working on the createEncounters() issue (#41). Should I remove the createEncounters() from my PR?

Reply:
Yeah maybe add a comment saying this PR is blocked by #41 and wait until that's merged in to get access to the method.